### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by edeyoung**
# Summary
Minor updates to the documentation including:
- fixing broken links
- updating information related to the validator
- fixing typos
- making phrasing consistent

# Testing Guidance
Visually review changes

### CODE REVIEW

File: docs/concepts.md [hunk 1]
```diff
@@ -52,18 +52,18 @@ third-party validators (e.g. HL7 FHIR Validator).
...
 for US Core and the Inferno ONC (g)(10) certification tests.
 
 ## Inferno Deployment
 
-A web host running one or more Inferno Test Kits. An example is the 
+An Inferno Deployment is a web host running one or more Inferno Test Kits. An example is the 
 [https://inferno.healthit.gov](https://inferno.healthit.gov) website.
 An individual Test Kit can also be run as an Inferno Deployment on 
-users' local machines without any additional
-configuration. An Inferno Deployment includes a web interface as well as
```
Comment (arscan) [inline docs/concepts.md:68]: Sometime fairly soon we'll want to introduce the concept of an 'Inferno Test Kit Platform' ([draft template here)](https://github.com/inferno-framework/inferno-platform-template).  Not quite yet though, but just reminding myself that we should update these once that is ready.

File: docs/index.md [hunk 2]
```diff
@@ -96,4 +91,4 @@ testing DSL.
   Inferno's Ruby API.
 
 ## Inferno Test Kits
-Visit the [Community Page](/inferno-core/available-test-kits) to view all available Test Kits.
+Visit the [Test Kits](https://inferno.healthit.gov/test-kits/) page to view all available Test Kits.
```
Comment (arscan) [inline docs/index.md:94]:
```suggestion
Visit the [Community Test Kits](/community/test-kits.html) page to view all available Test Kits.
```

Something that is a little confusing is that inferno.healthit.gov does not host all test kits.  Its just one 'Test Kit Platform' that ONC is paying hosting fees for.  It cannot be the only place you can run test kits though, as we aren't really in the SaaS/PaaS business.  So we maintain a separate list of test kits on this site.  Which, yeah, is redundant for the most part, but not quite completely, especially if more people use inferno to create testing solutions.

**Comment by bmath10** [comment]
Looks good, I can approve once you address Rob's comments!

**Comment by arscan** [comment]
Noticed that this is still hanging out here @edeyoung.  I just made one suggestion for your consideration.  Then once approved, we have the original author merge the PR.

**Comment by arscan** [comment]
I'll make a task to quickly run through this diff and apply the ones that are still applicable.  I think many are OBE, so attempting to do a rebase probably isn't worth the effort.  And I don't think @edeyoung should have to context switch back to this.

**Comment by mrnosal** [comment]
Closing this PR because changes were merged in from https://github.com/inferno-framework/inferno-framework.github.io/pull/60

**Pull request closed without merging**